### PR TITLE
Chore: fixing some `any`s/type assertions

### PR DESCRIPTION
**Pull request by ashharrison90**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: public/app/core/components/TagFilter/TagValue.tsx [hunk 1]
```diff
@@ -15,7 +15,7 @@ export class TagValue extends React.Component<Props, any> {
     this.onClick = this.onClick.bind(this);
   }
 
-  onClick(event: React.SyntheticEvent) {
+  onClick(event: React.SyntheticEvent<HTMLDivElement>) {
```
Comment (joshhunt) [inline public/app/core/components/TagFilter/TagValue.tsx:18]: maybe should update this to `event: React.MouseEvent<HTMLDivElement>`

**Review by joshhunt — APPROVED** [review]
good stuff 🚀

File: packages/grafana-data/src/utils/anyToNumber.ts [hunk 2]
```diff
@@ -5,7 +5,7 @@ import { toNumber } from 'lodash';
  *
  * @internal
  * */
-export function anyToNumber(value: any): number {
+export function anyToNumber(value: unknown): number {
   if (typeof value === 'number') {
     return value;
   }
```
Comment (joshhunt) [inline packages/grafana-data/src/utils/anyToNumber.ts:11]: basically exactly what `unknown` is for!

File: public/app/core/components/TagFilter/TagValue.tsx (hunk 1 above)
Comment (ashharrison90) [reply public/app/core/components/TagFilter/TagValue.tsx]: ah thanks, somehow missed that 😕

**Pull request merged**